docs/tutorials/bids_archive_tutorial.ipynb

```diff
@@ -0,0 +1,755 @@
+    "for inc in incrementals:\n",
+    "    newArchive.appendIncremental(inc)\n",
+    "\n",
+    "# Compare incrementals in original and new archive\n",
+    "for i in range(NUM_3D_IMAGES):\n",
+    "    assert archive.getIncremental(**entityFilterDict, sliceIndex=i) == \\\n",
+    "           newArchive.getIncremental(**entityFilterDict, sliceIndex=i)\n",
+    "    \n",
+    "# Note that trying to append to an incremental that conflicts with existing data\n",
+    "# (e.g., has the same metadata) will result in an error\n",
+    "try:\n",
+    "    newArchive.appendIncremental(incrementals[0])\n",
+    "except MetadataMismatchError as e:\n",
+    "    # When appending, the metadata will be checked to ensure an image isn't accidentally \n",
+    "    # being appended twice\n",
+    "    print(\"Append Error:\", e)\n",
+    "    \n",
+    "    # This behavior can be disabled at your own risk\n",
+    "    newArchive.appendIncremental(incrementals[0], validateAppend=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Cleanup\n",
+    "td.cleanup()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# BIDS Incremental: Creating Incremental\n",
+    "\n",
+    "A `BIDS Incremental` has two primary components:\n",
+    "1. A NIfTI image\n",
+    "2. A metadata dictionary storing information about the image.\n",
+    "\n",
+    "It also has a few other components that are used when the `BIDS Incremental` is written to disk, and may be used by you for other purposes. Those are:\n",
+    "1. The dataset description dictionary, which becomes the `dataset_description.json` in a BIDS Archive.\n",
+    "2. The README string, which becomes the `README` file in a BIDS archive.\n",
+    "3. The events dataframe, which becomes the `<file name entities>_events.tsv` file in a BIDS archive.\n",
+    "\n",
+    "To create a `BIDS Incremental`, only the image and the metadata dictionary are needed, and default versions of the other components are created if the `BIDS Incremental` is written to disk."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When reading from a BIDS-compliant dataset, all metadata\n",
+    "should already be present, and using BIDS Archive methods\n",
+    "to read the image and metadata is sufficient to create the\n",
+    "incremental."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Get the NIfTI image\n",
+    "imageFile = archive.getImages(subject='01', run=1)[0]\n",
+    "image = imageFile.get_image()\n",
+    "\n",
+    "# Get the metadata for the image\n",
+    "metadata = archive.getSidecarMetadata(imageFile)\n",
+    "\n",
+    "# Create the BIDS Incremental\n",
+    "incremental = BidsIncremental(image, metadata)\n",
```

Comment:
I got an error here:
MissingMetadataError: Image metadata missing required fields: ['subject', 'task', 'suffix', 'datatype']

Reply:
This is due to the change in getSidecarMetadata behavior -- I've revised the method name and default behavior to be a bit more clear (`getSidecarMetadata` -> `getImageMetadata` so that the `onlySidecar` parameter makes more sense). We can later add `getNiftiMetadata` and `getDatasetMetadata` to the overall BIDS Archive API.